### PR TITLE
test: get iftop output on test failure

### PR DESCRIPTION
**Pull request by aanm**
Signed-off-by: André Martins <andre@cilium.io>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/5884)

### CODE REVIEW

**Comment by aanm** [comment]
test-me-please

File: test/provision/helpers.bash [hunk 1]
```diff
@@ -79,4 +79,7 @@ function install_cfssl {
     sudo mv cfssl_linux-amd64 /usr/bin/cfssl
     chmod +x cfssljson_linux-amd64
     sudo mv cfssljson_linux-amd64 /usr/bin/cfssljson
-}
\ No newline at end of file
+}
+function install_iftop {
+    sudo apt-get install -y iftop
```
Comment (eloycoto) [inline test/provision/helpers.bash]: update the packer-ci--build and increment the build.

File: test/provision/helpers.bash [hunk 2]
```diff
@@ -79,4 +79,7 @@ function install_cfssl {
     sudo mv cfssl_linux-amd64 /usr/bin/cfssl
     chmod +x cfssljson_linux-amd64
     sudo mv cfssljson_linux-amd64 /usr/bin/cfssljson
-}
\ No newline at end of file
+}
+function install_iftop {
```
Comment (eloycoto) [inline test/provision/helpers.bash]: update the packer-ci--build and increment the build.

File: test/provision/helpers.bash (hunk 1 above)
Comment (eloycoto) [inline test/provision/helpers.bash:84]: update the packer-ci--build and increment the build.

**Review by eloycoto — CHANGES_REQUESTED** [review]

**Comment by aanm** [comment]
test-me-please

**Pull request closed without merging**